config/content/Sponsor.js

```diff
@@ -66,6 +67,38 @@ export default {
       alt: 'POLYGON LOGO',
       // width: '20%',
     },
+    {
+      src: 'https://res.cloudinary.com/dkpdi1wek/image/upload/v1708786350/download__2_-removebg-preview_1_dscm20.png',
+      name: 'MathWorks',
+      link: 'https://in.mathworks.com/',
+      alt: 'MathWorks LOGO',
+      // width: '20%',
+    },
+
+    {
+      src: 'https://res.cloudinary.com/dkpdi1wek/image/upload/v1708786569/download__3_-removebg-preview_otfa7p.png',
+      name: 'mongoDB',
+      link: 'https://www.mongodb.com/',
+      alt: 'mongoDB LOGO',
+      // width: '20%',
+    },
+
+    {
+      src: 'https://res.cloudinary.com/dkpdi1wek/image/upload/v1708786739/download__5_-removebg-preview_zslel5.png',
+      name: 'GoDaddy Registry',
+      link: 'https://in.godaddy.com/',
+      alt: ' GoDaddy LOGO',
+      // width: '20%',
+    },
+
+    {
+      src: 'https://res.cloudinary.com/dkpdi1wek/image/upload/v1708787057/download_6_sop5ix.png',
+      name: 'auth0',
+      link: 'https://auth0.com/',
+      alt: 'Auth0 LOGO',
+      // width: '20%',
+    },
+
```

Comment:
Where is ''flow"", its data is available in devfolio, add it.

Reply:
done